### ISSUE TITLE
A more streamlined interface

### ISSUE THREAD

**Issue by diaolulu1** [lllyasviel/Fooocus#740]
I suggest integrating the styles into the settings.
Line 189 of webui.py 

                performance_selection = gr.Radio(label='Performance', choices=['Speed', 'Quality'], value='Speed')
                style_selections = gr.Dropdown(label='Style',choices=legal_style_names,
                                               value=modules.path.default_styles,
                                               multiselect=True)
                aspect_ratios_selection = gr.Dropdown(label='Aspect Ratios', choices=list(aspect_ratios.keys()),
                                                   value=modules.path.default_aspect_ratio, info='width × height')

**Comment by barepixels** [comment]
With your mod:

Under the settings tab

Aspect Ratios pulldown menu looks great and WORKS
Style pulldown menu looks great BUT not working

Under the style tab (still exist)
Default checkboxes are still there and WORKS

I think you need to remove the style pulldown from Settings tab and convert the Style tab to pulldown.

**Comment by diaolulu1** [comment]
> 感谢敏神回复！

**Comment by diaolulu1** [comment]
> With your mod:
> 
> Under the settings tab
> 
> Aspect Ratios pulldown menu looks great and WORKS The style pulldown menu looks great BUT not working
> 
> ![setting](https://user-images.githubusercontent.com/92839033/277079955-f783923e-4d98-4656-b140-4a0eb3032ff7.png)
> 
> Under the style tab (still exists)
> 
> Default checkboxes are still there and WORKS
> 
> ![style](https://user-images.githubusercontent.com/92839033/277079983-3cd069ec-267f-4d8e-a0ae-afa73aaf246b.png)
> 
> I think you need to remove the style pulldown from the Setting tab and convert the Style tab to pulldown.
> 
> Would be cool if under Style tab to have 2 pull down. One for style and one for Artist Style
> 
> Come to think of it, it will be cleaner if we have 3 pull down in the Setting tab, Aspect ratio, Style, and Artist style. No style tab would be needed. Save room for future features.

Yes, I indeed have removed the Style tab, it has been replaced by the dropdown menu. It wouldn't work without removing it.

**Comment by diaolulu1** [comment]
Yes, I indeed have removed the Style tab, it has been replaced by the dropdown menu. It wouldn't work without removing it.

**Comment by barepixels** [comment]
Instruction for the new mod?
